test/unit/job_invocation_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_plugin_helper'
+
+describe JobInvocation do
+
+  let(:job_invocation) { FactoryGirl.build(:job_invocation) }
+  let(:template) { FactoryGirl.build(:job_template) }
+
+  context 'Able to be created' do
+    it { assert job_invocation.save }
+  end
+
+  context 'Requires targeting' do
+    before { job_invocation.targeting = nil }
+
+    it { refute job_invocation.save }
```

Comment:
`it { refute job_invocation.valid? }` should be enough, if you wan't to be extra cautious, you can verify `job_invocation.errors` on `:targeting` key